scripts/setup-socorro.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/sh
+
+# create DB if it does not exist
+# TODO handle DB not on localhost - could use setupdb for this
+su - postgres -c "psql breakpad -c ''" > /var/log/socorro/setupdb.log 2>&1
```

Comment:
Previously this stuff was run as part of the package install, and thus, implicitly as *root*; however, now that it has been split out into a separate script, `su`'s should likely be run via `sudo`.  Either that, or we mandate that the entire script be run via `sudo`.